pisa/utils/config_parser.py

```diff
@@ -528,13 +528,16 @@ def parse_param(config, section, selector, fullname, pname, value):
     return param
 
 
-def parse_pipeline_config(config):
+def parse_pipeline_config(config,skip_stages=None):
     """Parse pipeline config.
 
     Parameters
     ----------
     config : string or ConfigParser
 
+    skip_stages: None or List of Tuples identifying
+                 (service, stage) to skip in the parsing 
```

Comment:
only 4 spaces additional indent, don't line up with the colon above (see the developer's guide / Numpy docstring convention for any questions about docstring conventions in PISA)